src/Pos/Wallet/Web/Server/Methods.hs

```diff
@@ -136,7 +136,7 @@ walletApplication
     -> m Application
 walletApplication serv = do
     wsConn <- getWalletWebSockets
-    serv >>= return . upgradeApplicationWS wsConn . serve walletApi
+    upgradeApplicationWS wsConn . serve walletApi <$> serv
```

Comment:
Maybe it's very personal but `serv <&> upgradeApplicationWS wsConn . serve walletApi` may fit better for such cases.